lib/active_record/acts_as/querying.rb

```diff
@@ -28,8 +28,14 @@ def where!(opts, *rest)
 
     module ScopeForCreate
       def scope_for_create(attributes = nil)
+        unless acting_as?
+          return super(attributes) if Gem::Dependency.new('', '>= 5.2.1', '< 5.2.2').match?('', ActiveRecord.version)
```

Comment:
Can you change this to

```
if Gem::Dependency.new('', '>= 5.2.1', '< 5.2.2').match?('', ActiveRecord.version)
  return super(attributes)
else
  return super()
end
```

Looks cleaner IMO.
Thanks!

Reply:
Sure, no problem, pushed